enos/modules/vault_verify_performance_replication/main.tf

```diff
@@ -64,6 +64,21 @@ resource "enos_remote_exec" "verify_replication_on_primary" {
   }
 }
 
+resource "enos_remote_exec" "get_primary_replication_status" {
```

Comment:
Do we not need to put explicit depends_on clauses on this resource and the next to ensure that they run after the verify resource has completed?

Reply:
humm.. good point @mikebaum .  But we do want this resource to be run even when the previous one fails, but also don't want it to run before or in parallel.  probably that was the reason I originally called it inside the verification script.  wondering if I can call the inline after the script in the verify resource

Reply:
To resolve this issue, I renamed the script to correctly identify it with its purpose `get-replication-status`